Documentation/CRDs/ceph-block-pool-crd.md

```diff
@@ -206,7 +204,7 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
   require names that are incompatible with K8s resource names. These special pools can be configured
   by setting this `name` to override the name of the Ceph pool that is created instead of using the `metadata.name` for the pool.
   Only the following pool names are supported: `device_health_metrics`, `.nfs`, and `.mgr`. See the example
-  [builtin mgr pool](https://github.com/rook/rook/blob/{{ branchName }}/deploy/examples/pool-builtin-mgr.yaml).
+  [builtin mgr pool](https://github.com/rook/rook/blob/master/deploy/examples/pool-builtin-mgr.yaml).
```

Comment:
How can we replace the branch name to match the release branch? We don't want release docs to still point to github files in master.

Reply:
I see two ways on how we can approach this:
1. Add a Makefile target that is run during the release creation which does a search and replace of all `/master/` occurrences and sets them to the release branch. Meaning that every file pointing to a GitHub file/dir would be changed during that process and needs to be commited to the release branch.
2. Have a script auto replace the `/master/` when the documentation **is built** (I have something like this working, pushed to the branch). Doing this makes the links work when viewed on GitHub directly.

What do you think?

Reply:
Updating the links during the build makes sense, good to see that working now.